portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,285 @@
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col">
+                    <div class="card" style="width: 15rem;">
+                        <img class="card-img-top" src="/images/skiffel_logo.png" alt="Card image cap">
+                        <div class="card-body">
+                            <h5 class="card-title">Skiffel (Ios)</h5>
+                            <p class="card-text">
+                                Ios application for promoting and discovering social events.
+                            </p>
+                            <a href="https://apps.apple.com/bn/app/skiffel/id1475384987" target="_blank"
+                                class="btn btn-primary">View</a>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col">
+                    <div class="card" style="width: 15rem;">
+                        <img class="card-img-top" src="/images/skiffel_logo.png" alt="Card image cap">
+                        <div class="card-body">
+                            <h5 class="card-title">Skiffel (web)</h5>
+                            <p class="card-text">
+                                website to market skiffle.
+                            </p>
+                            <a href="https://skiffel.web.app/home" target="_blank" class="btn btn-primary">View</a>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col">
+                    <div class="card" style="width: 15rem;">
+                        <img class="card-img-top" src="/images/debt_defender.jpeg" alt="Card image cap">
+                        <div class="card-body">
+                            <h5 class="card-title">Debt defender</h5>
+                            <p class="card-text">
+                                Application built to help college students make better financial deciaions. Built furing
+                                a hackaton.
+                            </p>
+                            <a href="https://firestore-84244.firebaseapp.com/" target="_blank"
+                                class="btn btn-primary">View</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+
+    <section id="footer">
+        <h1 class="section-title">Contact</h1>
+        <footer class="bg-light">
+            <div class="row">
+
+                <div class="col"></div>
+                <div class="col">
+                    <ion-icon name="logo-linkedin"
+                        onclick="window.open('https://www.linkedin.com/in/tochukwu-ibe-ekeocha-949bb2170/')"></ion-icon>
+
+                </div>
+                <div class="col">
+                    <ion-icon name="logo-github" onclick="window.open('https://github.com/Tochukwuibe')"></ion-icon>
```

Comment:
This is unconventional and I wouldn't recommend using javascript to implement links this way. Since there's no `<a..>` tag, the browser doesn't offer any hints when hovering over this icon -- normally the mouse pointer changes to the `:hover` "pointer" style to indicate a clickable target.

A simpler way to implement these links is to just wrap the images, and specify a target:

```
<a href="https://..." target="_blank"> <ion-icon ... /> </a>
```

(Sometimes javascript is the only way to implement a certain link, and then I usually add a `cursor: pointer` css style to at least help the user know the page element is clickable)